fs/vfs/fs_ioctl.c

```diff
@@ -194,6 +194,12 @@ int nx_vioctl(int fd, int req, va_list ap)
                                nx_fcntl(fd, F_GETFL) & ~O_NONBLOCK);
               }
             break;
+          case FIOCLEX:
+            ret = nx_fcntl(fd, F_SETFD, FD_CLOEXEC);
```

Comment:
should we call F_GETFL and then or with FD_CLOEXEC? to avoid clear other flag we will add in the future.

Reply:
I will change them as follows

```
nx_fcntl(fd, F_GETFD) | FD_CLOEXEC
nx_fcntl(fd, F_GETFD) & ~FD_CLOEXEC
```

Reply:
I changed them in the commit https://github.com/apache/incubator-nuttx/pull/1800/commits/ac0c605c57e86445ac2aebcb6005b18bf268425a